pyproject.toml

```diff
@@ -397,6 +397,7 @@ show_deps = true
 skip = "*.lock,.direnv,.git,./docs/_freeze,docs/_freeze/**,*.svg,*.css,*.html,*.js"
 ignore-regex = '\b(i[if]f|I[IF]F|AFE)\b'
 builtin = "clear,rare,names"
+ignore-words-list = "tim"
```

Comment:
❓

Reply:
see above/slack channel screenshot, codespell complained that "Tim -> Time" in the two places the name is used

Reply:
Bleh :(